### ISSUE TITLE
directory

### ISSUE THREAD

**Issue by jfcarrero** [szweibel/Augur#3]
In which directory is augur. I installed apache and its in the home directory but when I type" python manage.py restart_db". I get "python: can't open file 'manage.py': [Errno 2] No such file or directory"

**Comment by szweibel** [comment]
Augur doesn't use Apache. Did you follow the installation instructions in the readme? Navigate to whichever directory you downloaded Augur into.

**Comment by jfcarrero** [comment]
Yes I did follow the directions in the readme.md file. I installed all the files in the requirements.txt . This is the directory the files are /usr/lib/Augur

Thank You for your quick response.

**Comment by jfcarrero** [comment]
Yes I did follow the directions in the readme.md file. I installed all the files in the requirements.txt . This is the directory the files are /usr/lib/Augur

Thank You for your quick response.

**Comment by szweibel** [comment]
So were you able to run the `python manage.py restart_db` command in that directory?

**Comment by jfcarrero** [comment]
I created a directory "sudo cd Augur" followed the instructions and it did the install. When I did the pytho manage.py restart I got this message "sqlalchemy.exc.OperationalError: (OperationalError) (1045, "Access denied for user '<USER>'@'localhost' (using password: YES)") None None" I'm trying to figure out how to change the password. I went to local:5000 and it's try to connect the message I get is "The server at local:5000 is taking too long to respond."

**Comment by szweibel** [comment]
did you try the command with `sudo`?

**Comment by jfcarrero** [comment]
I the installation as ROOT should I have done it as SU.

**Comment by szweibel** [comment]
If you installed it as ROOT I don't think there'd be a problem. Still, what you describe sounds like a problem with permissions.

**Comment by szweibel** [comment]
Reinstalling Augur myself, I realized what the problem was. In the installation instructions: 
"In settings.cfg, change the secret key to whatever you'd like, and **fill in the appropriate info for Mysql.**"
That part means that you should replace `<USER>` and `<PASSWORD>` with the root username and password you use for MySQL. I'll amend the instructions to make that clearer.

**Comment by jfcarrero** [comment]
I did make those changes to user and password before I contacted you. Where exactly do I create the Augur directory. I had it on my home directory. Could that be the issues with the permissions? When I went to local:5000 on the It seemed as it was trying to connect to the program but it would time out. I also tried it with the IPAddess:5000

**Comment by szweibel** [comment]
Directory location shouldn't matter if you're root.  So you put your username and password into the config file and it still gives the same error?
The browser timed out because the augur app isn't running.

**Comment by jfcarrero** [comment]
I got this error when I did a fresh Ubuntu OS and Augur install. "Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Shp5U0/MySQL-python/"

**Comment by jfcarrero** [comment]
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-5Q5n0F/MySQL-python/

**Comment by szweibel** [comment]
What step of the install were you at?

**Comment by jfcarrero** [comment]
python manage.py restart_db

**Comment by szweibel** [comment]
That's odd. I would try:
`sudo apt-get install python-dev`
`sudo apt-get install python-MySQLdb`

Also, what version of Python are you running?
`python --version`
